.github/workflows/uploadBinaries.yml

```diff
@@ -71,9 +71,9 @@ jobs:
               -DLLVM_ENABLE_TERMINFO=OFF \
               -DLLVM_PARALLEL_LINK_JOBS=1 \
               -DLLVM_TARGETS_TO_BUILD="host" \
-              -DLLVM_FORCE_ENABLE_STATS=${{ matrix.build_config.stats }}
+              -DLLVM_FORCE_ENABLE_STATS=${{ matrix.build_config.stats }} \
+              -DLLVM_ENABLE_ZSTD=OFF
           ninja
-          ninja check-llvm check-mlir
```

Comment:
Was this just `check-llvm` or also `check-mlir`? I usually run the latter and I didn't notice a problem with this version.

Reply:
Arguably if tests are failing the compiler isn't working so well, so it's good to fail the build.  Trick is to know about the issue before cutting a release, since these environments/toolchains aren't covered in our normal testing (yet?).

Reply:
I'll restore the tests. Part of why I added the artifact upload for non-tags is so that folks doing a release can kick off a run of this with a workflow dispatch on a branch and check the artifacts. Adding that manual step to the release process (and perhaps better yet adding it automatically somehow) makes the risk of the tests failing blocking publishing binaries a lot lower.